variables.tf

```diff
@@ -275,3 +275,9 @@ variable "consumer_quotas" {
   }))
   default = []
 }
+
+variable "default_network_tier" {
+  description = "Default Network Service Tier for resources created in this project. See https://cloud.google.com/network-tiers/docs/using-network-service-tiers and https://cloud.google.com/network-tiers."
```

Comment:
Clarify that if unset this value isn't updated at all.

Reply:
thanks, added the following text (bolded for emphasis here):

>Default Network Service Tier for resources created in this project. **If unset, the value will not be modified.** See https://cloud.google.com/network-tiers/docs/using-network-service-tiers and https://cloud.google.com/network-tiers.